beater/beater.go

```diff
@@ -43,17 +44,18 @@ func (bt *beater) Run(b *beat.Beat) error {
 		go bt.client.PublishAll(events)
 	}
 
-	server := newServer(bt.config, callback)
-	start(server, bt.config.SSL)
-	defer stop(server)
-
-	logp.Info("apm-server is running! Hit CTRL-C to stop it.")
-	// Blocks until service is shut down
-	<-bt.done
+	bt.server = newServer(bt.config, callback)
+	err = run(bt.server, bt.config.SSL)
+	logp.Err(err.Error())
```

Comment:
Comment seems to have disappeared during rebase. As we return the error, we don't need to log it at all. It will be printed out by the beat afterwards (if it is not nil)

Reply:
I checked the open comments and I think its fine. Error is always not nil, but the returned value to the beat might be nil, so there is a case for this log. thanks